docs/topics/db/queries.txt

```diff
@@ -884,6 +884,21 @@ object and has no return value. Example::
 
     e.delete()
 
+.. versionadded:: 1.9
```

Comment:
Try to make these annotations "self-contained" as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.